scipy/stats/tests/test_distributions.py

```diff
@@ -3852,6 +3852,61 @@ def test_cdf_ppf_sf_isf_tail(self, x, a, expected):
         isf = stats.dgamma.isf(expected, a)
         assert_allclose(isf, -x, rtol=5e-15)
 
+    @pytest.mark.parametrize("a, ref",
+        [(1.5, 2.0541199559354118267922018421759071394202272087878735),
+        (1.3, 1.9357296377121247644586320317632592032856887302320274),
+        (1.1, 1.7856502333412133408733521579693880954302277243835155)])
+    def test_entropy(self, a, ref):
+        #The reference values were calculated with mpmath:
+        #def entropy_dgamma(a):
+        #   def pdf(x):
+        #       A = 1 / (2 * mp.gamma(a))
+        #       B = mp.fabs(x) ** (a - 1)
+        #       C = mp.exp(-mp.fabs(x))
+        #       h = A * B * C
+        #       return h
+        #
+        #   return -mp.quad(lambda t: pdf(t) * mp.log(pdf(t)), [-mp.inf, mp.inf])
+        assert_allclose(stats.dgamma.entropy(a), ref)
+
+    @pytest.mark.parametrize("a, ref",
+    [(1e-100, -1.0e+100),
+    (1e-90, -1.0e+90),
+    (1e-80, -1.0e+80),
+    (1e-70, -1.0e+70),
+    (1e-60, -1.0e+60),
+    (1e-50, -1.0e+50),
+    (1e-40, -1.0e+40),
+    (1e-30, -1.0e+30),
+    (1e-20, -1.0e+20),
```

Comment:
```suggestion
```

Two tests with small `a` and two with large `a` should suffice.

Reply:
>Two tests with small a and two with large a should suffice.

I'm not too sure about that. When I was working on the Nakagami entropy, I noticed that there were sometimes values where it would fail, even though values higher and lower than it would pass. It's not a huge deal to me, but I would prefer keeping these values.

Reply:
In that case, maybe you should add values in between these?

Reply:
Ok, that sounds reasonable. What about the following test cases for example: `1e-8`, `1e-2`, `1`, `1e2`, `1e4`, `1e8`, `1e13`

Reply:
Ok, I've added some more test values.